index.html

```diff
@@ -0,0 +1,751 @@
+                        style="width: 630px; height: 426px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe></div>
+            </section>
+            <!-- Image -->
+            <section id="sec-image">
+                <h2 class="txt-header-2 txt-header-inv">Image</h2>
+                <!-- Normal Image -->
+                <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS_MFHGXBM_XVd_DKxWfyUTVAP5aOKkPRWuAA&usqp=CAU"
+                    alt="image" class="img">
+                <div class="code-frame"><iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28219%2C234%2C254%2C1%29&t=seti&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522img-location.png%2522%250A%2520%2520alt%253D%2522image%2522%2520class%253D%2522img%2522%253E"
+                        style="width: 378px; height: 223px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe></div>
+                <h3>Responsive Image</h3>
+                <img src="https://www.iamabiker.com/wp-content/uploads/2018/04/Royal-Enfield-Thunderbird-350X-HD-wallpapers-1.jpg"
+                    alt="image" class="img img-res">
+                <div class="code-frame"><iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28219%2C234%2C254%2C1%29&t=seti&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522img-location.png%2522%250A%2520%2520alt%253D%2522image%2522%2520class%253D%2522img%2520img-res%2522%253E"
+                        style="width: 437px; height: 223px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe></div>
+                <h3>Round Image</h3>
+                <img src="./images/logo.png" alt="image" class="img img-res img-rd">
+                <div class="code-frame"><iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28219%2C234%2C254%2C1%29&t=seti&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522img-location.png%2522%250A%2520%2520alt%253D%2522image%2522%2520class%253D%2522img%2520img-res%2520img-rd%2522%253E"
+                        style="width: 496px; height: 223px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
inline style can be avoided.
Also border:0; transform: scale(1); overflow:hidden; is repeated in each iframe so can be added in a class.

Reply:
sure, this approach sounds good. Thanks!